### PR TITLE
Disable parallel tests on travis

### PR DESCRIPTION
**Pull request by devshorts**

### CODE REVIEW

File: core/src/test/java/suites/BuildVerificationTestSuite.java [hunk 1]
```diff
@@ -38,7 +37,6 @@
         QueueRepositoryTester.class,
         ApiAuthenticationTests.class,
         SystemPropTests.class,
-        ParallelWorkerTests.class,
```
Comment (jakeswenson) [inline core/src/test/java/suites/BuildVerificationTestSuite.java:41]: don't remove this, this should always be there

Comment (devshorts) [inline core/src/test/java/suites/BuildVerificationTestSuite.java:41]: gotcha, it'll be xcluded with the junit filter?

Comment (jakeswenson) [inline core/src/test/java/suites/BuildVerificationTestSuite.java:41]: yes see the ExcludeCategories above? as long as test marked there have one of those they will be excluded

Comment (devshorts) [inline core/src/test/java/suites/BuildVerificationTestSuite.java:41]: :+1:

Comment (devshorts) [inline core/src/test/java/suites/BuildVerificationTestSuite.java:41]: done

**Comment by jakeswenson** [comment]
:heavy_check_mark: LGTM

**Pull request merged**